openshift/rh-che.app.yaml

```diff
@@ -0,0 +1,316 @@
+kind: Template
+apiVersion: v1
+metadata:
+  labels:
+    app: che
+  name: che
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    labels:
+      app: che
+    name: che
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: che
+    name: che-host
+  spec:
+    ports:
+    - name: http
+      port: 8080
+      protocol: TCP
+      targetPort: 8080
+    selector:
+      app: che
+- apiVersion: v1
+  kind: RoleBinding
```

Comment:
I am not sure that is needed anymore. We don't use `che` ServiceAccount anymore to create workspace so we should not require it to be admin.

Reply:
fixed

Reply:
looks like we need this role binding:

2017-11-08 19:13:43,576[ost-startStop-1]  [INFO ] [o.e.c.a.m.s.r.RecipeLoader 74]       - Recipes initialization finished
--
  | 2017-11-08 19:13:43,618[ted-scheduler-2]  [INFO ] [ockerAbandonedResourcesCleaner 126]  - List containers registered in the api: []
  | 2017-11-08 19:13:43,641[ost-startStop-1]  [WARN ] [p.s.AdminPermissionInitializer 68]   - Admin admin not found yet.
  | 2017-11-08 19:13:43,712[ost-startStop-1]  [ERROR] [o.a.c.c.C.[.[.[/wsmaster] 4733]      - Exception sending context initialized event to listener instance of class org.eclipse.che.inject.CheBootstrap
  | com.google.inject.CreationException: Unable to create injector, see the following errors:
  | 1) Error in custom provider, io.fabric8.kubernetes.client.KubernetesClientException: Failure executing: GET at: https://kubernetes.default.svc/api/v1/namespaces/latest-template/services/che-host. Message: Forbidden!Configured service account doesn't have access. Service account may have been revoked..